mm2src/mm2_main/src/rpc/dispatcher/dispatcher.rs

```diff
@@ -261,6 +263,8 @@ async fn rpc_task_dispatcher(
         "withdraw::status" => handle_mmrpc(ctx, request, withdraw_status).await,
         "withdraw::user_action" => handle_mmrpc(ctx, request, withdraw_user_action).await,
         #[cfg(not(target_arch = "wasm32"))]
+        "z_storage::min_height" => handle_mmrpc(ctx, request, get_minimum_header_from_cache).await,
```

Comment:
@CharlVS I see that @borngraced added this before the SoW was agreed upon, do you think this will be needed at all in the GUIs? If not it can be removed. This endpoint is to get the block height the wallet started syncing from anytime after activation instead of saving it in GUIs if the GUIs will need it.

Reply:
@shamardy It won't be needed and can safely be removed.